### ISSUE TITLE
No pod spec in chatsdk

### ISSUE THREAD

**Issue by DrDRoarty** [chat-sdk/chat-sdk-ios#42]
I notice there is no pod spec file in ChatSDK in latest release, any reason?

Best

Dave

![screen shot 2017-04-26 at 17 05 20](https://cloud.githubusercontent.com/assets/6763547/25444355/b163225e-2aa2-11e7-8972-7a23f0ae013e.png)

**Comment by jimijon** [comment]
You have to pod them in individually. The pod specs are in the other 4 directories

**Comment by DrDRoarty** [comment]
Yep, worked that out already, should have updated the thread, my bad.
Thanks for the response my friend  -  appreciate it
 have a good day

**Comment by bensmiley** [comment]
Yes, we decided to include the pods individually in this release.

**Comment by DrDRoarty** [comment]
Hi Ben

Yes I worked it out however all the documentation on the site explaining how to integrate into your own app and your video will now be hard to follow for the newer developers 

Does that make sense?

Just thought I would flag it up

Best

Kind Regards
Dave Roarty

**Comment by bensmiley** [comment]
We've updated the documentation but we will have to update the videos... Thanks.
